maps/us-healthcare-system-capacity/js/main.js

```diff
@@ -84,10 +84,11 @@ function onMapChange() {
   resetFillPaintStyle("hrr-fill");
   resetLinePaintStyle("hrr-line");
   resetCirclePaintStyle("facility-circle");
-  if (usePerCapita(indicator)) {
+  if (allowTypeChange(indicator)) {
```

Comment:
There was an issue with using `usePerCapita`, so broke the disable check into it's own function.